__tests__/app.test.js

```diff
@@ -67,3 +67,43 @@ describe('GET /api/topics', () => {
       });
   });
 });
+
+describe('GET /api/articles/:article_id', () => {
+  test('200: Responds with an article object with the correct properties', () => {
+    return request(app)
+      .get('/api/articles/1')
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toEqual(
+          expect.objectContaining({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: 1,
```

Comment:
Nice! Well done for making sure the test is still flexible but that we definitely have received back the correct article regardless of what the article info is!